packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table/use-row-data/generate-property-rows-from-entity/generate-property-row-recursively.ts

```diff
@@ -78,15 +78,14 @@ export const generatePropertyRowRecursively = (
           [...propertyKeyChain, subPropertyTypeBaseUri],
           rootEntityAndSubgraph,
           requiredPropertyTypes,
-          depth + 1,
```

Comment:
I don't think this change was intentional?

Reply:
Thank you so much for caching it, yeah it was not intentional